treeherder/log_parser/parsers.py

```diff
@@ -388,6 +388,7 @@ class ErrorParser(ParserBase):
         r"|^The web-page 'stop build' button was pressed"
         r"|.*\.js: line \d+, col \d+, Error -"
         r"|^\[taskcluster\] Error:"
+        r"|^\[[\w-]+:(?:error|exception)+\]"
```

Comment:
(Or just repeatedly force push to the PR as you iterate, and Travis will do it for you :-))